prow/monitoring/prometheus_operator_crd/Makefile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: apply
+
+# Commit hash for repo github.com/operator-framework/community-operators
+COMMIT ?= 42131df7167ec0b264c892c1f3c49ba9a72142da
+
+download:
+	@echo "downloading prometheus CRDs ..."
+	curl -O -s https://raw.githubusercontent.com/operator-framework/community-operators/$(COMMIT)/upstream-community-operators/prometheus/alertmanager.crd.yaml
```

Comment:
Rather than checking in these files you can reference it using a `git_repository()` rule (or maybe new_git_repository?):

https://github.com/kubernetes/test-infra/blob/4dbd419a2ea8d53036d447c3b63f92a3306fce33/WORKSPACE#L20-L26

Inject a BUILD file that makes it export all its files:

https://github.com/kubernetes/test-infra/blob/4dbd419a2ea8d53036d447c3b63f92a3306fce33/WORKSPACE#L207-L213

and then you should be able to reference it in the k8s_object rule - https://github.com/bazelbuild/rules_k8s#k8s_object


new_git_repository: https://github.com/bazelbuild/bazel/blob/f718a4249739f63e3a0c6e008faa3c7d47fa30f9/tools/build_defs/repo/git.bzl#L204